pkg/api/v1/schema.go

```diff
@@ -697,7 +697,7 @@ type Interface struct {
 	// TODO:(ihar) switch to enums once opengen-api supports them. See: https://github.com/kubernetes/kube-openapi/issues/51
 	Model string `json:"model,omitempty"`
 	// BindingMethod specifies the method which will be used to connect the interface to the guest.
-	// Defaults to Bridge.
+	// Defaults to HostBridge.
```

Comment:
What does this change in defaults imply? For what I see, `DefaultNetworkInterface` still creates a `bridge` network. Is it something else?